app/src/main/java/de/westnordost/streetcomplete/quests/charging_station_operator/AddChargingStationOperator.kt

```diff
@@ -13,7 +13,11 @@ class AddChargingStationOperator : OsmFilterQuestType<String>() {
     override val icon = R.drawable.ic_quest_car_charger
     override val isDeleteElementEnabled = true
 
-    override fun getTitle(tags: Map<String, String>) = R.string.quest_charging_station_operator_title
+    override fun getTitle(tags: Map<String, String>): Int =
+        if (tags.containsAnyKey("name", "brand"))
+            R.string.quest_charging_station_name_operator_title
+        else
+            R.string.quest_charging_station_operator_title
```

Comment:
This quest is not asked for charging station that have a name, brand or operator

Reply:
and also should not be, of course

Reply:
Is it not the case that they could have a brand or operator and need a name set (e.g. at a service station)?

For example this company seem to name them (effectively based on their location), while the brand/operator would be Pod Point:
https://charge.pod-point.com/

Maybe this wants to be a separate issue to discuss it?

Reply:
It could, but any of the three is usually enough to identify what kind of charging station it is, i.e. derive what socket types it supports etc.
As with ATMs, it's not that well standardized what exactly should go into brand, what into operator and what into name.

Reply:
Yeah it would be fine for the how many sockets quest. I was referring to this comment of yours:

> This quest is not asked for charging station that have a name, brand or operator

Given this is the AddChargingStationOperator quest we're discussing it in.

> As with ATMs, it's not that well standardized what exactly should go into brand, what into operator, what into network and what into name.

NSI is working hard to improve on that...